src/views/Stake/Stake.jsx

```diff
@@ -423,31 +426,32 @@ function Stake() {
                     <TableRow>
                       <TableCell>
                         <Box className="ohm-pairs">
-                          <BondLogo bond={ohm_frax}></BondLogo>
-                          <Typography>OHM-FRAX</Typography>
+                          <BondLogo bond={{ bondIconSvg: OhmLusdImg, isLP: true }}></BondLogo>
+                          <Typography>OHM-LUSD</Typography>
                         </Box>
                       </TableCell>
-                      <TableCell align="left">{fraxData && trim(fraxData.apy, 1)}%</TableCell>
+                      <TableCell align="left">{lusdData && trim(lusdData.apy, 1)}%</TableCell>
                       <TableCell align="left">
-                        {fraxData &&
-                          fraxData.tvl &&
+                        {lusdData &&
+                          lusdData.tvl &&
                           new Intl.NumberFormat("en-US", {
                             style: "currency",
                             currency: "USD",
                             maximumFractionDigits: 0,
                             minimumFractionDigits: 0,
-                          }).format(fraxData.tvl)}
+                          }).format(lusdData.tvl)}
                       </TableCell>
-                      <TableCell align="left"> {(fraxData && fraxData.balance) || 0} LP </TableCell>
+                      <TableCell align="left"> {(lusdData && trim(lusdData.balance, 2)) || 0} LP </TableCell>
```

Comment:
stuff like this makes me realize that we need to work better to generalize this section. kek.